doc/jenkins.rst

```diff
@@ -0,0 +1,136 @@
+Integration with Jenkins
+========================
+
+`Universum` requires no special integration with `Jenkins`, except for one thing. It is usually launched
+as one long step in a single build stage. Because of that, the whole `Universum` log is printed as a plain text
+without navigation anchors.
+
+Compare with these:
+
+* When ran locally, all Universum build step logs are stored to separate files instead of printing their
+  output to console
+* When launched by TeamCity, `Unviersum` uses `service messages
+  <https://www.jetbrains.com/help/teamcity/service-messages.html>`__ to increase log readability
+
+To simplify navigating long logs and finding relevant information on build results, we provide a user-friendly
+interactive log with collapsible blocks and other features.
```

Comment:
Here you can mention that the log is usable even outside of Jenkins, but for Jenkins it provides missing functionalty.